designcompose/src/main/java/com/android/designcompose/squoosh/SquooshTreeBuilder.kt

```diff
@@ -192,22 +181,24 @@ internal fun resolveVariantsRecursively(
 
         // Do we have an override style? This is style data which we should apply to the final style
         // even if we're swapping out our view definition for a variant.
-        if (componentInfo.overrides.isPresent) {
-            val overrides = componentInfo.overrides.get()
-            if (overrides.style.isPresent) {
-                overrideStyle = overrides.style.get()
+        if (v.componentInfo.hasOverrides()) {
+            val overrides = v.componentInfo.overrides
+            if (overrides.hasStyle()) {
+                overrideStyle = overrides.style
             }
         }
```

Comment:
Can simplify this to 
```suggestion
        overrideStyle = v.componentInfoOrNull?.overridesOrNull?.styleOrNull
```

Reply:
Done